### PR TITLE
add section contacts.html and modals.html (location, franchise)

### PR DESCRIPTION
**Pull request by NataliaNazarenko**
Привіт. Зробила html розмітку секції contacts і двох модалок (location, franchise). Але модалки закоментувала, щоб поки не заважали.

### CODE REVIEW

**Review by ruudyaa — CHANGES_REQUESTED** [review]
Send all the required changes into DM:
- class=“visuali-hidden”
- classes list, link - unnecessary (the styles for links and lists are reset in the reset file)
- “contacts_geo” 
- the element h3 must not appear as a descendant of the <address> element
- the text content of the element time was not in the required format: The literal did not satisfy the time-DateTime format.
- location.html and franchise.html do not have a closing icon:
<use href=“./images/icons.svg#“></use>

**Comment by NataliaNazarenko** [comment]
Привіт. Пофіксила html. Додала стилі для картки. Кнопки і модалки ще без стилів. Також є проблема з карткою для мобілки.

**Review by ruudyaa — CHANGES_REQUESTED** [review]
conflict with icons

**Comment by NataliaNazarenko** [comment]
Привіт. Пофіксила css секції контакти. Прописала стартові стилі для кнопок.

File: src/partials/contacts.html [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+      </li>
+
+      <li class="contacts__item">
+        <p class="contacts__pre-title">CAFE</p>
+        <h3 class="contacts__title">New York</h3>
+        <address class="contacts__addresses">
+          <a
+            href="https://goo.gl/maps/vUbYZenNEvYyMwox6"
+            target="_blank"
+            rel="noopener noreferrer norofollow"
```
Comment (ruudyaa) [inline src/partials/contacts.html]: norofollow - nofollow

File: src/partials/contacts.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+      </li>
+
+      <li class="contacts__item">
+        <p class="contacts__pre-title">FOOD TRUCK</p>
+        <h3 class="contacts__title">Los Angeles</h3>
+        <address class="contacts__addresses">
+          <a
+            href="https://goo.gl/maps/2uWRQVcKfBqWiVA78"
+            target="_blank"
+            rel="noopener noreferrer norofollow"
```
Comment (ruudyaa) [inline src/partials/contacts.html]: norofollow - nofollow

File: src/partials/contacts.html [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+    <ul class="contacts__list">
+      <li class="contacts__item">
+        <p class="contacts__pre-title">CAFE</p>
+        <h3 class="contacts__title">Chicago</h3>
+        <address class="contacts__addresses">
+          <a
+            href="https://goo.gl/maps/X3x532GnqmV6juVT9"
+            target="_blank"
+            rel="noopener noreferrer norofollow"
```
Comment (ruudyaa) [inline src/partials/contacts.html]: norofollow - nofollow

File: src/sass/layout/_contacts.scss [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+}
+
+.contacts__item {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  max-width: 280px;
+  max-height: 412px;
+  padding: 40px 36px 24px 32px;
+  background-color: #ffffff;
```
Comment (ruudyaa) [inline src/sass/layout/_contacts.scss]: color should be written as variable

File: src/partials/contacts.html (hunk 3 above)
Comment (NataliaNazarenko) [reply src/partials/contacts.html]: fixed norofollow - nofollow

File: src/partials/contacts.html (hunk 2 above)
Comment (NataliaNazarenko) [reply src/partials/contacts.html]: fixed norofollow - nofollow

File: src/partials/contacts.html (hunk 1 above)
Comment (NataliaNazarenko) [reply src/partials/contacts.html]: fixed norofollow - nofollow

File: src/sass/layout/_contacts.scss [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+  display: flex;
+  width: max-content;
+  align-items: center;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: calc(14 / 12);
+  letter-spacing: 0.04em;
+  color: getColor('theme-black');
+  background-color: getColor('yellow-main');
+  border-radius: 6px;
```
Comment (ruudyaa) [inline src/sass/layout/_contacts.scss]: border-radius: 4px; - for mobile screen and 480px

File: src/sass/layout/_contacts.scss (hunk 4 above)
Comment (NataliaNazarenko) [reply src/sass/layout/_contacts.scss]: add and fixed

File: src/sass/layout/_contacts.scss (hunk 5 above)
Comment (NataliaNazarenko) [reply src/sass/layout/_contacts.scss]: fixed

File: src/sass/layout/_contacts.scss [hunk 6]
```diff
@@ -0,0 +1,298 @@
...
+  & + span {
+    font-weight: 500;
+    color: getColor('theme-grey-2');
+  }
+}
+
+.contacts__connection {
+  max-width: 130px;
+  max-height: 44px;
+
```
Comment (ruudyaa) [inline src/sass/layout/_contacts.scss:206]: @include tablet {
  width: 151px;
  height: 46px;
}

File: src/sass/layout/_contacts.scss [hunk 7]
```diff
@@ -0,0 +1,298 @@
...
+    color: getColor('theme-black');
+  }
+
+  &:not(:last-child) {
+    margin-bottom: 6px;
+  }
+
+  @include tablet {
+    font-size: 14px;
+    line-height: calc(26 / 14);
```
Comment (ruudyaa) [inline src/sass/layout/_contacts.scss]: line-height: calc(21 / 14);

File: src/sass/layout/_contacts.scss (hunk 6 above)
Comment (NataliaNazarenko) [reply src/sass/layout/_contacts.scss:206]: add

File: src/sass/layout/_contacts.scss (hunk 7 above)
Comment (NataliaNazarenko) [reply src/sass/layout/_contacts.scss]: fixed

File: src/sass/layout/_contacts.scss [hunk 8]
```diff
@@ -0,0 +1,298 @@
...
+.button__group {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+
+  @include tablet {
+    flex-direction: row;
+    justify-content: center;
+    align-items: stretch;
+    margin-right: 16px;
```
Comment (ruudyaa) [inline src/sass/layout/_contacts.scss]: margin is for the whole group, not for one button.

File: src/sass/layout/_contacts.scss [hunk 9]
```diff
@@ -0,0 +1,298 @@
...
+  border: none;
+  border-radius: 10px;
+  cursor: pointer;
+
+  &.button__text {
+    font-weight: 600;
+    color: getColor('theme-black');
+  }
+
+  &:not(:last-child) {
```
Comment (ruudyaa) [inline src/sass/layout/_contacts.scss]: only for mobile screens and 480px

File: src/sass/layout/_contacts.scss (hunk 8 above)
Comment (NataliaNazarenko) [reply src/sass/layout/_contacts.scss]: fixed

File: src/sass/layout/_contacts.scss [hunk 10]
```diff
@@ -0,0 +1,298 @@
...
+    margin-right: 16px;
+  }
+
+  @include desktop {
+    flex-direction: row;
+    justify-content: center;
+  }
+}
+
+.button {
```
Comment (ruudyaa) [inline src/sass/layout/_contacts.scss:263]: + font-family: inherit;

File: src/partials/modals/franchise.html [hunk 11]
```diff
@@ -0,0 +1,39 @@
...
+      Due to the overwhelming volume of franchise requests we are not accepting new franchisee
+      applications at this time. Once the application process resumes, we will contact you for
+      further information. Please fill out the contact information below and we will reach out for
+      additional information. We appreciate your patience.
+    </p>
+
+    <form class="form">
+      <div class="form__wrap">
+        <label for="name"></label>
+        <input class="form__input" name="name" id="name" placeholder="Name" />
```
Comment (ruudyaa) [inline src/partials/modals/franchise.html]: Should be added pattern for name validation + required.

File: src/partials/modals/franchise.html [hunk 12]
```diff
@@ -0,0 +1,39 @@
...
+
+        <label for="tel"></label>
+        <input class="form__input" name="tel" id="tel" placeholder="Phone" />
```
Comment (ruudyaa) [inline src/partials/modals/franchise.html]: Should be added pattern for phone number validation + required.

File: src/partials/modals/franchise.html [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+
+        <label for="email"></label>
+        <input class="form__input" name="email" id="email" autocomplete="off" placeholder="Email" />
```
Comment (ruudyaa) [inline src/partials/modals/franchise.html]: Should be added pattern for email validation + required.

File: src/sass/layout/_contacts.scss (hunk 9 above)
Comment (NataliaNazarenko) [reply src/sass/layout/_contacts.scss]: fixed

File: src/sass/layout/_contacts.scss (hunk 10 above)
Comment (NataliaNazarenko) [reply src/sass/layout/_contacts.scss:263]: add

**Review by ruudyaa — CHANGES_REQUESTED** [review]
Fix, please, all the things mentioned in the comments.

File: src/partials/modals/franchise.html (hunk 11 above)
Comment (NataliaNazarenko) [reply src/partials/modals/franchise.html]: add

File: src/partials/modals/franchise.html (hunk 12 above)
Comment (NataliaNazarenko) [reply src/partials/modals/franchise.html]: add

File: src/partials/modals/franchise.html (hunk 13 above)
Comment (NataliaNazarenko) [reply src/partials/modals/franchise.html]: add

**Pull request closed without merging**